cuqi/experimental/mcmc/_cwmh.py

```diff
@@ -199,23 +199,3 @@ def tune(self, skip_len, update_count):
         # Update the scale parameter
         self.scale = np.minimum(scale_temp, np.ones(self.dim))
         self._scale_temp = scale_temp
-
-    def get_state(self):
```

Comment:
really great how much redundancy were removed (the definitions of get_state and set_state that were everywhere)!